examples/oic.r.acl-example2.json

```diff
@@ -29,61 +29,6 @@
           }
         ]
       }
-    ],
```

Comment:
(This comment applies to the "aclist" -> "aces" example above)

I see the acl2.raml file example omits the "aces" tag over the array of ace2 objects, but this example includes the "aces" tag.  I don't see the "aces" tag in the .json for either acl2 or ace2, so I'm suspecting that it's meant to omitted and is just in this example by error, but please clarify!

Also, the "subjectuuid" should be "subject"... I'm guessing you didn't have time to scrub this example yet except to remove the second "aces2" example which was definitely wrong.